### DISCUSSION TITLE
API : Any way to interrupt and stop current job?

### DISCUSSION THREAD

**Discussion by caniyabanci76** [comfyanonymous/ComfyUI#1514] [Q&A] [answered]
Apart from killing the server, is there anyway to stop the current job of an API script that is in progress?

**Comment by IPv6** [comment, marked as answer]
Any chance this is solved? //

UPD: nevermind, there is "/interrupt"

**Reply by PratyushChauhan** [reply]
hey, i wanted to ask if there is a way to cancel a job by providing a job_id
